.markdownlint.json

```diff
@@ -0,0 +1,12 @@
+{
+    "default": true,
+    "MD004": {"style": "asterisk"},
+    "MD007": { "indent": 4 },
+    "MD013": false,
+    "MD024": { "siblings_only": true },
+    "MD025": false,
+    "MD033": false,
+    "MD034": false,
+    "no-hard-tabs": false,
+    "whitespace": false
+}
```

Comment:
@julienrbrt Perhaps add something in the Readme that explains how to have code editors pick it up? I have found [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) for VSCode, which I don't really use.

Reply:
The extension for VSCode will just work using the `markdownlint.json` without extra configuration, as the config file in the root of the project.

Reply:
True, but I had to Google that, and install the extension to confirm it before writing my comment. It would be good to make it clear in the Readme too.